src/db/User/user.ts

```diff
@@ -45,5 +54,5 @@ export async function editUserInDb(userId: ObjectId, newUserFields: Partial<User
 	await _connection.close();
 	if (userToUpdate.acknowledged === false) throw "Could not update User";
```

Comment:
Should this throw a `DBOperationError`? If not, should we create another custom error type for situations like these, or does it not really matter?

Reply:
yeah for this I would say `DbOperationError`. didn't change them all since I figured could be handled by a PR in the future, and we need to get authentication working before the next milestone